test/functional/apps/dashboard/dashboard_time_picker.js

```diff
@@ -67,15 +67,15 @@ export default function ({ getService, getPageObjects }) {
         name: 'saved search',
         fields: ['bytes', 'agent'],
       });
-      // Current data grid loads 100 rows per page by default with inspect button and time range
-      await dashboardExpect.savedSearchRowCountFromLegacyTable(100);
+      // DefaultDiscoverTable loads 10 rows initially
+      await dashboardExpect.rowCountFromDefaultDiscoverTable(10);
```

Comment:
nit: does it makes sense to export pageSize as a constant and use that here to avoid future updates?

Reply:
It does. I will add that.